spec/services/proofing/aamva/request/verification_request_spec.rb

```diff
@@ -33,7 +33,7 @@
 
   describe '#body' do
     it 'should be a request body' do
-      expect(subject.body).to eq(AamvaFixtures.verification_request)
+      expect(subject.body + "\n").to eq(AamvaFixtures.verification_request)
```

Comment:
"This is disgusting!," you might protest again. And again, I agree.

I am going to file a followup story or five, because:

1. As @lmgeorge pointed out, expecting absolute string equality when comparing XML documents isn't quite right.
2. `add_optional_element` is somehow leading to ugly mis-indentation.
3. I tried to clean it up with [REXML::Formatters::Pretty](https://ruby-doc.org/3.3.5/gems/rexml/REXML/Formatters/Pretty.html) but somehow made an even bigger mess that complained it wasn't valid XML. And if we're going to format our XML nicely, I want us to be sending it to AAMVA that way, not _just_ cleaning it up for a test.